src/routes/Home.tsx

```diff
@@ -105,10 +104,10 @@ const $MessageLine2 = styled.span`
   font-weight: 900;
   font-size: 55px;
   text-transform: uppercase;
-  color: #59c9e6;
+  color: #3e99b0;
 `
 
-const $MessageLine3 = styled.span`
+const $MessageLine3 = styled.h1`
```

Comment:
The fact that this is called `$MessageLine3` makes me think h1 doesn't make sense for it... Or is this used for the main heading?

Reply:
as far as i remember, line 3 holds the highest font-size. and that's why